docs/dev-guides/3-distributing-xrpackage.md

```diff
@@ -3,44 +3,75 @@ id: distributing-xrpackage
 
-Ethereum has a native cryptocurrency ('digital money') called <a href="https://ethereum.org/eth/" target="_blank" rel="noopener noreferrer">Ether (ETH)</a>.
+**Note:** The uniqueness of your XRPackage's `name` is enforced, as this is what will identify your package on the registry.
 
-Ethereum gives ownership and interoperability of your packages -- you will always have a complete sense of ownership for anything you create and can provably sell anything you own.
+### What is IPFS?
 
-## What is the Registry?
+IPFS is the <a href="https://ipfs.io/" target="_blank" rel="noopener noreferrer">InterPlanetary File System</a>, which is a publicly distributed hypermedia file-sharing protocol system. IPFS uses distributed hash tables to store and find files by referencing a _cryptographic hash_ which encrypts the file contents to assign a unique data identifier for your XRPackage file.
 
-The Registry is a simple database that maps user-friendly names to the unique long hashes produced by the IPFS. This means you can give your package the name `chicken` instead of its hash `QmdVXHq9TWdcDeHQmYAWQpCp8pnXx96Jc9PxJVdUVstTB9` (see <a href="https://xrpackage.org/inspect.html?p=chicken" target="_blank" rel="noopener noreferrer">here</a>) making it more discoverable and user-friendly.
+IPFS is also an open source, _decentralized_ system - it uses a peer-to-peer protocol to spread data across the network of computers instead of storing your files on a single centralized server. This means you can publish your packages yourself, without needing to rely on your own (costly), or someone else's (unreliable or potentially unsecure) server!
+
+Put simply, IPFS is a decentralized hosting solution for publicly storing and sharing your XRPackage files.
+
+#### What is the Webaverse Package Registry?
+
+The Webaverse Package Registry is a public online package repository of user-created published packages hosted within Webaverse's site. Published packages map user-friendly names to the unique long hashes produced by the IPFS. This means you can give your package the name `chicken` instead of its hash `QmdVXHq9TWdcDeHQmYAWQpCp8pnXx96Jc9PxJVdUVstTB9` (see <a href="https://xrpackage.org/inspect.html?p=chicken" target="_blank" rel="noopener noreferrer">here</a>) making it more discoverable and user-friendly.
 
 You can browse the XRPackage Registry at <a href="https://xrpackage.org/browse.html" target="_blank" rel="noopener noreferrer">xrpackage.org</a>.
 
-## Publishing to IPFS
+#### Using the Webaverse Package Registry
 
-**Note: The publish functionality in the XRPackage CLI currently only publishes to the Rinkeby testnet. You will need to have a sufficient ETH balance in order to publish packages. You can get free Rinkeby testnet ETH <a href="https://faucet.rinkeby.io/" target="_blank" rel="noopener noreferrer">at the faucet</a>.**
+It's really simple to use the Registry to publish your XRPackage &mdash; just 2 commands!
```

Comment:
We should mention it's also possible to publish a package entirely from the registry site.